src/braket/parametric/free_parameter_expression.py

```diff
@@ -39,6 +53,7 @@ def __init__(self, expression: Union[FreeParameterExpression, Number, Expr, str]
 
         Args:
             expression (Union[FreeParameterExpression, Number, Expr, str]): The expression to use.
+            _type (Optional[ClassicalType]): type of the expression
```

Comment:
A reference to the types here would help clarify the use. These are all used for QASM nodes but the var name, someone may assume that it is a generic type.  

What changes in OQpy `0.3.1` are utilizing this?